app/javascript/components/icon/styles.scss

```diff
@@ -0,0 +1,11 @@
+.c-icon {
+  width: 30px;
```

Comment:
Are all your icons this size [30 , 20]? weird... I imagine that you will have more sizes in the future, I'm ok for now, but be careful, I'm watching...